src/cargo/core/compiler/timings/report.rs

```diff
@@ -52,35 +49,28 @@ impl SectionData {
 /// Save HTML report to disk.
 pub(super) fn report_html(
```

Comment:
Seems like this should be renamed to `write_html` since it isn't doing any reporting but rendering what it is told

Reply:
Renamed.